net-misc/econnman/econnman-1.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4..7} )
+
+inherit python-any-r1
```

Comment:
python-any-r1 is unsuitable for anything that gets installed. The docs explicitly say that.

Reply:
Yes, I read it and chose it for this one. 
>...that need Python during build-time only and therefore need not to be strongly bound to a particular Python implementation. 

I dont know if Im thinking into this too much or too little, but I fail to see how changing python eclass would make this installation any different...
```
# equery f econnman
 * Searching for econnman ...
 * Contents of net-misc/econnman-1.1:
/usr
/usr/bin
/usr/bin/econnman-bin
/usr/share
/usr/share/applications
/usr/share/applications/econnman-agent.desktop
/usr/share/applications/econnman.desktop
/usr/share/doc
/usr/share/doc/econnman-1.1
/usr/share/doc/econnman-1.1/AUTHORS
/usr/share/doc/econnman-1.1/README.bz2
/usr/share/econnman
/usr/share/econnman/theme
/usr/share/econnman/theme/default.edj
```

I understand installing python scripts under site-packages and calling python-exec is a cleaner way, but this works as-is, and is not overly complicated.

One thing that does come to my mind is to run `python_fix_shebang` on *econnman-bin* since some people might have python2 as default. 
But what do you think, about eclass and shebang?

Reply:
Installing a Python script is also 'Python stuff'. That's precisely why python-single-r1 exists.